#!/usr/bin/env python
PACKAGE = "laserscan_virtualizer"

from dynamic_reconfigure.parameter_generator_catkin import *
import math

gen = ParameterGenerator()

gen.add("angle_min", double_t, 0, "Minimum angle of the output scan", -2.36, -math.pi, math.pi)
gen.add("angle_max", double_t, 0, "Maximum angle of the output scan", 2.36, -math.pi, math.pi)
gen.add("angle_increment", double_t, 0, "Angle increment of the output scan", 0.0058, 0, math.pi)
gen.add("time_increment", double_t, 0, "Time increment of the output scan", 0.0, 0, 1)
gen.add("scan_time", double_t, 0, "Scan time of the output scan", 0.033333333, 0, 1)
gen.add("range_min", double_t, 0, "Range min of the output scan", 0.45, 0, 50)
gen.add("range_max", double_t, 0, "Range max of the output scan", 25.0, 0, 50)

exit(gen.generate(PACKAGE, "laserscan_virtualizer", "laserscan_virtualizer"))
